ui/packages/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.view.js

```diff
@@ -50,7 +50,6 @@ module.exports = Marionette.LayoutView.extend({
       this.triggerCloseDropdown
     )
     this.generateSearchFormSelector()
-    this.generateSearchSettings()
```

Comment:
why was this deleted?

Reply:
This is to remove the confusing settings option in the searches. It used to exist under the `User another search form` option
<img width="611" alt="Screen Shot 2019-04-16 at 2 09 42 PM" src="https://user-images.githubusercontent.com/23177404/56240691-5b2ffd80-6051-11e9-96dd-e7305c268ec9.png">